tests/integration/targets/vmware_host_iscsi/tasks/reset_iscsi_config_tasks.yml

```diff
@@ -0,0 +1,41 @@
+# Test code for the vmware_host_logbundle module.
+# Copyright: (c) 2020, sky-joker <sky.jokerxx@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: reset iSCSI config tasks
+  block:
+    - vmware_host_iscsi:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: no
+        esxi_hostname: "{{ esxi1 }}"
+        iscsi_config:
+          alias: ''
+          vmhba_name: "{{ vmhba_name }}"
+        state: present
+
+    - vmware_host_iscsi:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: no
+        esxi_hostname: "{{ esxi1 }}"
+        iscsi_config:
+          port_bind:
+            - vmk0
+          vmhba_name: "{{ vmhba_name }}"
+          send_target:
+            address: 100.64.0.1
+          static_target:
+            iscsi_name: iqn.2011-08.com.xxxxxxx:as6104t-8c3e9d.target001
+            address: 100.64.0.2
+        state: absent
+
+    - vmware_host_iscsi:
```

Comment:
If I understand correctly, this reset the host to its initial state. Can you call the task from `always` section. This way, we will be sure we restore the test environment after a failure.

Reply:
@sky-joker can you address this comment?

Reply:
Thank you for your comment.  
I've removed a `rest_iscsi_config_tasks.yml`, and I have moved the tests that weren't duplicated in `reset_iscsi_config_tasks.yml` to `iscsi_module_test_tasks.ym`

https://github.com/ansible-collections/vmware/blob/027b3ff3457644cf72c43ab2f07588007f9dabff/tests/integration/targets/vmware_host_iscsi/tasks/iscsi_module_test_tasks.yml#L840-L891

https://github.com/ansible-collections/vmware/blob/027b3ff3457644cf72c43ab2f07588007f9dabff/tests/integration/targets/vmware_host_iscsi/tasks/iscsi_module_test_tasks.yml#L1063-L1105

I'm sorry if the answer is not what you expected.